packages/lib-classifier/src/components/Classifier/components/Layout/Layout.js

```diff
@@ -1,24 +1,23 @@
+import { observer } from 'mobx-react'
 import PropTypes from 'prop-types'
```

Comment:
```suggestion
```

Reply:
Have we forgotten to define prop types for `Layout`?

Reply:
There used to be a `layout` prop, but it was deleted in this PR:
```
Layout.propTypes = {
  layout: PropTypes.string
}
```